lib/thunder/pages/notifications_pages.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:lemmy_api_client/v3.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import 'package:thunder/inbox/bloc/inbox_bloc.dart';
+import 'package:thunder/inbox/widgets/inbox_replies_view.dart';
+import 'package:thunder/post/bloc/post_bloc.dart';
+
+/// A page for displaying the result of reply notifications
+class NotificationsReplyPage extends StatelessWidget {
+  final List<CommentReplyView> replies;
+
+  const NotificationsReplyPage({super.key, required this.replies});
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final AppLocalizations l10n = AppLocalizations.of(context)!;
+
+    return MultiBlocProvider(
+      providers: [
+        BlocProvider.value(value: InboxBloc()),
+        BlocProvider.value(value: PostBloc()),
+      ],
+      child: Container(
+        color: theme.colorScheme.background,
+        child: SafeArea(
+          top: false,
+          child: CustomScrollView(
+            slivers: <Widget>[
+              SliverAppBar(
+                flexibleSpace: const FlexibleSpaceBar(titlePadding: EdgeInsets.zero),
+                pinned: true,
+                title: ListTile(
+                  title: Text(l10n.inbox, style: theme.textTheme.titleLarge),
+                  subtitle: Text(l10n.reply(replies.length)),
+                ),
+              ),
+              SliverToBoxAdapter(
+                child: Material(
+                  child: Transform(

```

Comment:
Just curious but what is the `Transform` for?

Reply:
I found that there was a space between the `SliverAppBar` and the `InboxRepliesView`. If I use another widget, there's no space, so I don't think it's related to the app bar. But there is also no padding defined in the `InboxRepliesView` and seemingly nothing above the first divider/reply. I worked on tracking it down for a while but got stuck, hence this. Let me know if you have any other ideas!

Reply:
Is it possible that the `Column` in `InboxRepliesView` is causing this? Try to add this to the `Column`:

```dart
crossAxisAlignment: CrossAxisAlignment.start,
mainAxisAlignment: MainAxisAlignment.start,
```

Reply:
Just tried, that doesn't help. 😞  Here's what it looks like for reference.

https://github.com/thunder-app/thunder/assets/7417301/cf75ba12-e450-47a6-8e66-2bccd3a63673

Reply:
Could you try adding `padding: EdgeInsets.zero` to ListView.builder in `InboxRepliesView`? 

> By default, [ListView](https://api.flutter.dev/flutter/widgets/ListView-class.html) will automatically pad the list's scrollable extremities to avoid partial obstructions indicated by [MediaQuery](https://api.flutter.dev/flutter/widgets/MediaQuery-class.html)'s padding. To avoid this behavior, override with a zero [padding](https://api.flutter.dev/flutter/widgets/BoxScrollView/padding.html) property.

It looks like `ListView` may be adding padding to the top. Usually, we would use `SliverList` for lists in `CustomScrollView` but we can't do this in this case since it's being shared. For reference: https://api.flutter.dev/flutter/widgets/ListView-class.html

Reply:
That did it! And it doesn't mess up the Inbox page either. Thank you so much for tracking that down!